src/Package/Util/Header/Version.php

```diff
@@ -90,14 +90,14 @@ public function fileHasVersionMacro($fname)
     public function getVersionFromHeader()
     {
         $headers = glob($this->package->getSourceDir().DIRECTORY_SEPARATOR.'*.h');
-        $pat = ',define\s+'.$this->macroName.'\s+"(.*)",i';
+        $pat = ',define\s+'.$this->macroName.'\s+(".*"|.*\b),i';
```

Comment:
The current regex (`/define\s+foo\s+(".*"|.*\b)/`) matches the following forms:
  1. `define foo "hello"`,
  2. `define foo hello`.

I guess it's fair enough. And nop, this is not possible to match only an open quote without the closing quote as @stof said. However, @duncan3dc, if you woud like something stricter, then simply replace `".*"` by `"[^"]+". This is not exactly the same because `".*"` will match `""` while `"[^"]+"` requires at least one character inside double quotes (which, from my point of view, is better).